### PR TITLE
Add errors with explanation for the removed options warn and pathToMake

### PR DESCRIPTION
**Pull request by Razzeee**
This should report helpful errors when using the warn or pathToMake option.

I tried to test it, but still get this error:

image

### CODE REVIEW

**Comment by Razzeee** [comment]
Found the problem will squash soon

File: index.js [hunk 1]
```diff
@@ -75,8 +75,10 @@ function compilerErrorToString(err, pathToElm) {
       default:
         return ("Error attempting to run Elm compiler \"" + pathToElm + "\":\n" + err);
     }
+  } else if ((typeof err === "object") && (typeof err.message === "string")) {
+    return (JSON.stringify(err.message));
```
Comment (eeue56) [inline index.js]: no need for `()` around `JSON.stringify(err.message)` here

File: index.js [hunk 2]
```diff
@@ -75,8 +75,10 @@ function compilerErrorToString(err, pathToElm) {
...
   } else {
-    return ("Exception thrown when attempting to run Elm compiler " + JSON.stringify(pathToElm) + ":\n");
+    return ("Exception thrown when attempting to run Elm compiler " + JSON.stringify(pathToElm));
```
Comment (eeue56) [inline index.js]: same as above

Comment (Razzeee) [reply index.js]: Fixed

File: index.js (hunk 1 above)
Comment (Razzeee) [reply index.js]: Fixed

**Comment by rtfeldman** [comment]
Love it, thank you @Razzeee!

**Comment by rtfeldman** [comment]
Published as 5.0.1.

**Comment by Razzeee** [comment]
Thanks for the quick feedback loop :)

**Pull request merged**